modules/programs/tmux.nix

```diff
@@ -109,7 +109,7 @@ let
         }
     )];
 
-    home.file.".tmux.conf".text = ''
+    xdg.configFile."tmux/tmux.conf".text = ''
```

Comment:
I think this whole change should be gated behind a new state version. See https://github.com/nix-community/home-manager/pull/1919/files#diff-f1f119fe42c1c025c898ab5a4b5063f7867d8f3d327111f1b029ef13a29fe1dcR128 for an exaple of how to do this.